src/Multiselect.vue

```diff
@@ -288,89 +344,120 @@ export default {
      */
     showNoOptions: {
       type: Boolean,
-      default: true
+      default: true,
     },
     showNoResults: {
       type: Boolean,
-      default: true
+      default: true,
     },
     tabindex: {
       type: Number,
-      default: 0
-    }
+      default: 0,
+    },
+  },
+  watch: {
+    value: {
+      // the watch will look to any changes in value to adjust selectedOptions for adding and removing values
+      immediate: true,
+      handler(newVal, oldVal) {
+        if (newVal && oldVal) {
+          if (newVal.length > oldVal.length) {
+            this.selectedOptions.push(newVal[newVal.length - 1]);
+          } else {
+            this.selectedOptions = this.selectedOptions.filter((option) =>
+              newVal.includes(option)
+            );
+          }
+        }
+      },
+    },
+    selectedOptions: {
+      deep: true,
+      handler: function (newVal) {
+        this.$emit("addtag", newVal);
```

Comment:
I don't understand why the emitted event should be `addtag`, not the already existing `input` events

```suggestion
        this.$emit("input", newVal);
```